main.py

```diff
@@ -35,10 +34,9 @@ def main():
     browser.sleep(60)
     browser.close()
 
-
 def read_data():
     try:
-        data = pd.read_csv('data.csv')
+        data = sheet_reader()
```

Comment:
Could you make the Google Sheet optional? I.e. add a global flag to control the choice between using a plain .csv and using a Google Sheet. For backwards compatibility, and I'd rather not make the use of Google Sheets a prerequisite. Some people may prefer Excel or Libre Office or something else on their own machine.